Helper/Data.php

```diff
@@ -328,14 +380,30 @@ public function optInSubscriber($email, $data = [])
      */
     public function cronSubscribeAll($list)
     {
+        $subscribers = [];
         foreach ($list as $batch) {
-            $response = $this->callApi('contact', $batch, 'POST');
-            if (!array_key_exists('message', $response) ||
-                array_key_exists('message', $response) && $response['message'] !== 'OK') {
+            // Subscribe customers to each website separately.
+            foreach ($this->getWebsiteIds() as $websiteId) {
+                if (! $this->isCronEnabledForWebsite($websiteId)) {
+                    continue;
+                }
+
+                // Filter subscribers by website ID.
+                $subscribers = array_filter($batch, function ($ar) use ($websiteId) {
```

Comment:
Try not to use abstract variable names. A developer might not understand what `$ar` stands for. A better variable name would be `$subscriber`.

Reply:
Are there more occurrences? I remember only $ar being abstract